bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_ggbjj_Mgg80-13000

```diff
@@ -0,0 +1,70 @@
+(run){
+  EVENTS 1000;
+  EVENT_MODE HepMC;
+
+  ME_SIGNAL_GENERATOR Comix Internal;
+  EVENT_GENERATION_MODE Unweighted;
+
+  BEAM_1 2212; BEAM_ENERGY_1 6500.;
+  BEAM_2 2212; BEAM_ENERGY_2 6500.;
+
+  PDF_LIBRARY     LHAPDFSherpa;
+  PDF_SET         NNPDF30_nnlo_as_0118;
+  PDF_SET_VERSION 0;
+  PDF_GRID_PATH   PDFsets;
+
+  CSS_EW_MODE 1;
+  ME_QED Off;
+}(run)
+
+(processes){
+#  Process 21 21 -> 22 22;
+#  Scales VAR{Abs2(p[2]+p[3])};
+#  ME_Generator Internal;
+#  Loop_Generator gg_yy;
+#  End process;
+
```

Comment:
Please remove the above commented out lines

Reply:
Sure I shall clean it up.